pandas/core/internals/blocks.py

```diff
@@ -932,7 +932,7 @@ def setitem(self, indexer, value):
         # value must be storable at this moment
         if is_extension_array_dtype(getattr(value, "dtype", None)):
             # We need to be careful not to allow through strings that
-            #  can be parsed to EADtypes
```

Comment:
this is a pattern i use intentionally to distinguish between

```
# this is a a very long comment that is split into
#  multiple lines
# this is a separate comment that happens to be nearby
```

Reply:
Got it, reverted the whole file.